src/Microsoft.PowerShell.Commands.Utility/commands/utility/Send-MailMessage.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// Implementation for the Send-MailMessage command.
     /// </summary>
+    [Obsolete("This cmdlet only supports sending mail from insecure SMTP servers. While there is no immediate replacement available in PowerShell, we recommend you do not use Send-MailMessage at this time.")]
```

Comment:
Please add test to validate.

Reply:
WarningVariables does not capture this warning.

Reply:
Try using `powershell.invoke` and then check the streams.

Reply:
It would be nice to have a xUnit test suite for _all public_ APIs to have API contract.